package/lib/src/beamer_back_button_dispatcher.dart

```diff
@@ -69,7 +69,7 @@ class BeamerChildBackButtonDispatcher extends ChildBackButtonDispatcher {
         fallbackToBeamBack = parent.fallbackToBeamBack,
         super(parent);
 
-  /// A [BeamerDelegate] that belongs to the same [Router]/[Beamer] as this.
+  /// A [BeamerDelegate] that belongs to the same [Router]/`Beamer` as this.
```

Comment:
I believe we should leave `[Beamer]`, to reference the [Beamer Widget](https://github.com/slovnicki/beamer/blob/master/package/lib/src/beamer.dart) which was intended.
(same goes for line 18 above)